src/orion/benchmark/task/base.py

```diff
@@ -46,12 +45,16 @@ def __call__(self, *args, **kwargs):
     @property
     def max_trials(self):
         """Return the max number of trials to run for the"""
-        return self.trials_num
+        return self._max_trials
+
+    @property
+    def trial_num(self) -> int:
```

Comment:
Why add it and deprecate it right away?

Reply:
True, I'll just remove that attribute instead.